### PR TITLE
AT_18.05_002 Profile Page> dashboard  My view link

### PR DESCRIPTION
**Pull request by Kseniyaza**
https://trello.com/c/vDVQXVPA/1020-at1805002-profile-page-open-dashboard-with-my-view-link

### CODE REVIEW

File: cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress"/>
+
+describe('KZ-profilePageOpenDashboardMyViewLink.cy.js', () => {
```
Comment (Rom-T) [inline cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js:3]: Please name the describe as the name of your file.

File: cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const expectedMyViewItems = [
+        'Test',
+    ];
```
Comment (Rom-T) [inline cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js:6]: Please move `data` in the fixtures

File: cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (DariaMas7) [inline cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js:38]: please delete all of the empty lines we don't need them

**Review by DariaMas7 — COMMENTED** [review]
Its better to write your code in already existing file for this US 
profilePageOpenDashboardMyViewLink.cy

File: cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress"/>
+
+describe('KZ-profilePageOpenDashboardMyViewLink.cy.js', () => {
+    const expectedMyViewItems = [
+        'Test',
+    ];
+    beforeEach('Create folder', function () {
+        cy.get('a[href="/view/all/newJob"]')
```
Comment (Rom-T) [inline cypress/e2e/profilePageOpenDashboardWithMyViewLink.cy.js:8]: This line does nothing. Please delete it.

**Review by SvetlanaVet — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Review by Rom-T — CHANGES_REQUESTED** [review]
Close please PR

**Pull request closed without merging**